paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -105,13 +106,7 @@ internal fun SectionElementUI(
         val controller = element.controller
 
         val error by controller.error.asLiveData().observeAsState(null)
-        val sectionErrorString =
-            error?.let {
-                stringResource(
-                    it.errorMessage,
-                    stringResource(it.errorFieldLabel)
-                )
-            }
+        val sectionErrorString = error?.getMessage(LocalContext.current)
```

Comment:
Is there any way to do this without passing in the context?  

Could we at least limit it to resources?

Reply:
We can generate the message here instead of the `FieldError`.
Using `stringResource` it would look ugly because of the varargs:
```
error?.let {
                it.formatArgs?.let { args ->
                    stringResource(
                        it.errorMessage,
                        args
                    )
                } ?: stringResource(it.errorMessage)
            }
```

Reply:
^ It actually has to be this way otherwise the array is considered a single argument